content/language/word-list/cripple.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cripple, crippled, crippleware
+---
+
+**Reasoning:** 
+
+The terms cripple, crippled, and crippleware are sometimes used in writing about software or computer systems.
```

Comment:
```suggestion
The terms cripple, crippled, and crippleware are sometimes used in writing about software or computer systems because people are stupid.
```